lib/js/index.js

```diff
@@ -291,17 +331,30 @@ class DateTimePicker extends Events {
   setTime(time) {
     const m = moment(time);
     const hour = m.format('HH');
+    const minutes = m.format('mm');
     const hourAsInt = parseInt(hour, 10) % 12;
+    const minuteAsInt = parseInt(minutes, 10);
     const oldActiveHours = this.$(`.c-datepicker__clock__hours .${this.options.styles.clockNum}--active`);
+    const oldActiveMinutes = this.$(`.c-datepicker__clock__minutes .${this.options.styles.clockNum}--active`);
 
     this.$('.js-date-hours').innerText = hour;
+    this.$('.js-date-minutes').innerText = minutes;
 
     if (oldActiveHours) {
       oldActiveHours.classList.remove(`${this.options.styles.clockNum}--active`);
     }
+
+    if (oldActiveMinutes) {
+      oldActiveMinutes.classList.remove(`${this.options.styles.clockNum}--active`);
+    }
+
     this.$(`.c-datepicker__clock__hours .${this.options.styles.clockNum}[data-number="${hourAsInt}"]`)
       .classList.add(`${this.options.styles.clockNum}--active`);
+    this.$(`.c-datepicker__clock__minutes .${this.options.styles.clockNum}[data-number="${minuteAsInt}"]`)
```

Comment:
I found a bug in testing - if the minutes from the datepicker's `default` time are not on the clock faces, this throws:

```
index.js:353 Uncaught TypeError: Cannot read property 'classList' of undefined
    at DateTimePicker.setTime (index.js:353)
    at DateTimePicker.open (index.js:92)
    at HTMLAnchorElement.<anonymous>
```

You can test this by adding `default: moment()` to the Datepicker options. If the current minutes is not a multiple of 5, you'll hit this error.